docker/docker-init.sh

```diff
@@ -15,12 +15,10 @@ then
     echo "${SSH_PRIVATE_KEY}" | tr -d '\r' | ssh-add -
 fi
 
-# space-separated list of balena CLI commands (filled in through `sed`
-# in a Dockerfile RUN instruction)
-CLI_CMDS="help"
-
-# try to determine if command is a balena CLI command
-if [[ " ${CLI_CMDS} " =~ " ${1} " ]]
+# treat the provided command as a balena CLI arg...
+# 1. if the first word is present in cli_cmds matching word boundaries
+# 2. OR if the first character is a hyphen (eg. -h or --help)
```

Comment:
Added support to detect global flags if the first character is a hyphen. Shouldn't be required but nice to have I think.